models/order-alternative/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+
+export * as OrderFromCartDraft from './order-from-cart-draft';
+export * as OrderFromQuoteDraft from './order-from-quote-draft';
+export * as Order from '.';
+
+export const random = RestModelBuilder;
+export { default as presets } from './presets';
+export { default as draftPresets } from './order-from-cart-draft/presets';
+
+export * from './types';
+export * as constants from './constants';
+
+export const OrderRest = {
+  random: RestModelBuilder,
+};
+export const OrderGraphql = {
+  random: GraphqlModelBuilder,
+};
```

Comment:
Just a thought: we don't need a barrel export file. We could also go `orders/rest`, `orders/graphql` and `orders/draft` and import as `import { Order as OrderGraphQl } from '@commercetools/test-data/order/graphql` for instance.

Reply:
That's an interesting alternative we must evaluate, although I personally find it easier for the package to export its assets form a main entry point.
Open for discussion.

Reply:
Yes, it's just minor in the end and depends on the consumer I think.